test/createComponent.spec.js

```diff
@@ -46,4 +47,20 @@ describe('createComponent', function () {
     expect(myComponentInstance.myCustomFunction()).to.be.equal('foo');
     expect(myComponentInstance.render()).to.be.equal(null);
   });
+
+  it('throws an error if no render method is defined', function () {
+    const noRenderSpec = {};
+
+    expect(createComponent.bind(null, noRenderSpec))
+      .to.throw('Render is a required method.');
+  });
+
+  it('throws an error if render is not a function', function () {
+    const badRenderSpec = {
+      render: 'it is a string moron'
```

Comment:
We don't want words like this in our documentation. I have issued a new PR so that our builds fail for sensitive words like this: https://github.com/Travix-International/frint/pull/27